test/session_test.py

```diff
@@ -84,7 +84,7 @@ def test_temp_works_without_currently_active_session(self):
             assigned_site = shopify.ShopifyResource.site
 
         self.assertEqual('https://testshop.myshopify.com/admin/api/unstable', assigned_site)
-        self.assertEqual('https://None/admin/api/unstable', shopify.ShopifyResource.site)
+        self.assertEqual('https://none/admin/api/unstable', shopify.ShopifyResource.site)
```

Comment:
I am guessing this is downcased now because the parsing of the host downcases the host?

Reply:
Yeah, I'm not exactly sure what usecase this particular test is after, but I imagine the case change doesn't matter because this isn't a real URL anyway.